src/interfaces/index.ts

```diff
@@ -0,0 +1,13 @@
+export * from './decorator-options/expose-option.interface';
+export * from './decorator-options/exclude-option.interface';
+export * from './decorator-options/transform-option.interface';
+export * from './decorator-options/type-discriminator-descriptor.interface';
+export * from './decorator-options/type-option.interface';
```

Comment:
I suggest renaming these files to `.options` (with an `s`).

Reply:
Done.